### PR TITLE
Add suport to PHP 8.0 and PHP 8.1

### PR DESCRIPTION
**Pull request by lucassmacedo**
Add suport to PHP 8.0 and PHP 8.1

### CODE REVIEW

**Comment by dbpolito** [comment]
@lucassmacedo out Dockerfiles are auto generated running `kool run template`, as described in the README: https://github.com/kool-dev/docker-php-oci8#update-images-with-templates

Is there any change in the images? Other than the base image? If so, just changing the https://github.com/kool-dev/docker-php-oci8/blob/main/fwd-template.json and running `kool run template` should do the job

**Comment by lucassmacedo** [comment]
@dbpolito i'm doing a new pull request, there's no change besides the base image.

**Comment by lucassmacedo** [comment]
Done.

**Comment by dbpolito** [comment]
@lucassmacedo do you mind adding the non prod ones too? to follow the standard

**Review by lucassmacedo — COMMENTED** [review]
Add suport to PHP 8.0 and PHP 8.1 in fwd-template file

**Comment by dbpolito** [comment]
also, you need to cleanup the 8.2 image, and the README

**Comment by lucassmacedo** [comment]
Check now please.

File: fwd-template.json [hunk 1]
```diff
@@ -24,6 +24,30 @@
                     "path": "template/Dockerfile"
                 }
             ]
+        },
+        {
+            "name": "8.0-nginx-oci8-prod",
```
Comment (dbpolito) [inline fwd-template.json:29]: you need to add here without prod too... for both versions, and run template again

Comment (lucassmacedo) [reply fwd-template.json:29]: Check now please.

**Review by dbpolito — APPROVED** [review]

**Pull request merged**